ts/receiver/contentMessage.ts

```diff
@@ -480,6 +481,11 @@ export async function innerHandleSwarmContentMessage({
     window.log.info('innerHandleSwarmContentMessage');
 
     const content = SignalService.Content.decode(new Uint8Array(contentDecrypted));
+    if (!shouldProcessContentMessage(envelope, content, false)) {
+      window.log.info('innerHandleSwarmContentMessage: dropping invalid content message');
```

Comment:
Don't you want to log the timestamp or an identifier here for debugging?

related to https://github.com/session-foundation/session-desktop/pull/69#discussion_r1953781456